pkg/termination/handler_test.go

```diff
@@ -50,13 +50,18 @@ var _ = Describe("Handler Suite", func() {
 		httpHandler = nil
 		nodeName = "test-node"
 		httpHandler = newMockHTTPHandler(notFoundFunc)
+		stop = nil
```

Comment:
I agree this should be the fix for unit flakes, re: https://github.com/openshift/cluster-api-provider-aws/pull/340#issuecomment-661748943